apps/miniapp/pages/index.tsx

```diff
@@ -1,20 +1,66 @@
+import { useState } from 'react'
 import { useIntl } from '@core/next/intl'
-import { PageContent, PageHeader, PageWrapper } from '@miniapp/domains/common/components/BaseLayout'
-import { useOidcAuth } from '@miniapp/domains/common/utils/oidcAuth'
-import { useOrganization } from '@miniapp/domains/common/utils/organization'
+import dayjs from 'dayjs'
+import { Button } from '@condo/domains/common/components/Button'
+import getConfig from 'next/config'
 import { Typography } from 'antd'
 import { get } from 'lodash'
 import Head from 'next/head'
+import faker from 'faker'
+import { PageContent, PageHeader, PageWrapper } from '@miniapp/domains/common/components/BaseLayout'
+import { useOidcAuth } from '@miniapp/domains/common/utils/oidcAuth'
+import { useOrganization } from '@miniapp/domains/common/utils/organization'
+
+const { publicRuntimeConfig: { condoUrl } } = getConfig()
 
 const IndexPage = () => {
     const intl = useIntl()
     const auth = useOidcAuth()
     const organizationData = useOrganization()
 
     const PageTitleMsg = intl.formatMessage({ id: 'pages.index.PageTitle' })
+    const TasksTitleMsg = intl.formatMessage({ id: 'pages.index.tasks.Title' })
+    const TasksAddMsg = intl.formatMessage({ id: 'pages.index.tasks.Add' })
+    const TasksUpdateMsg = intl.formatMessage({ id: 'pages.index.tasks.Update' })
     const userDataMsg = intl.formatMessage({ id: 'UserData' })
     const organizationDataMsg = intl.formatMessage({ id: 'OrganizationData' })
 
+    const [latestTaskRecord, setLatestTaskRecord] = useState({})
+
+    const postMessageToAddTask = () => {
```

Comment:
It’s not a part of this pr. @sitozzz check it please

Reply:
@pahaz it's just a demonstration of progress component API